services/frontend/src/conf.js

```diff
@@ -29,3 +29,9 @@ export const builtAt = process.env.REACT_APP_BUILT_AT || ''
 export const serviceProvider = process.env.REACT_APP_SERVICE_PROVIDER
   ? process.env.REACT_APP_SERVICE_PROVIDER.toLowerCase()
   : 'toska'
+
+// Variable that can be used to disable the language center view, by default the view is enabled
+export const languageCenterViewEnabled =
+  process.env.REACT_APP_LANGUAGE_CENTER_VIEW_ENABLED == null
```

Comment:
Should this work also with?
`const languageCenterViewEnabled = !process.env.LANGUAGE_CENTER_VIEW_ENABLED
  ? true
  : process.env.LANGUAGE_CENTER_VIEW_ENABLED === 'true'`

Reply:
Same as :arrow_up: